packages/js/src/helpers/sortArrayOfObjects.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Sorts array of objects alphabetically based on the object text content.
+ * E.g. [ { name: "dogs" }, { name: "cats" } ] ->  [ { name: "cats" }, { name: "dogs" } ]
+ *
+ * @param {Object[]} array An array objects.
+ *
+ * @returns {Object[]} The alphabetically sorted array of objects.
+ */
+export const sortArrayOfObjects = ( array ) => {
+	return array.sort( ( a, b ) => {
+		const nameA = a.name.toLowerCase();
+		const nameB = b.name.toLowerCase();
+		if ( nameA < nameB ) {
+			return -1;
+		}
+		if ( nameA > nameB ) {
+			return 1;
+		}
+		return 0;
+	} );
+};
```

Comment:
I took this method of sorting array of objects alphabetically from [this documentation page](https://developer.mozilla.org/en-US/docs/web/javascript/reference/global_objects/array/sort) of `.sort()` method. If you know a short/better approach, feel free to leave a comment :).